kubernetes/cray-sysmgmt-health/templates/prometheus/disk-space/disk-space.yaml

```diff
@@ -54,6 +54,27 @@ spec:
       for: 1h
       labels:
         severity: critical
+    - alert: ContainersVolumeFull
+      annotations:
+        message: 'Available space on mountpoint /rootfs/var/lib/containers is less than "{{`{{ $value | printf "%.2f" }}`}}" % '
+      expr: node_filesystem_avail_bytes{mountpoint="/rootfs/var/lib/containers",job="cray-sysmgmt-health-ceph-node-exporter"} / node_filesystem_size_bytes{mountpoint="/rootfs/var/lib/containers",job="cray-sysmgmt-health-ceph-node-exporter"} < 0.05
+      for: 1h
+      labels:
+        severity: critical
+    - alert: KubeletVolumeFull
+      annotations:
+        message: 'Available space on mountpoint /var/lib/kubelet is less than "{{`{{ $value | printf "%.2f" }}`}}" % '
+      expr: node_filesystem_avail_bytes{mountpoint="/var/lib/kubelet",job="node-exporter"} / node_filesystem_size_bytes{mountpoint="/var/lib/kubelet",job="node-exporter"} < 0.05
+      for: 1h
+      labels:
+        severity: critical
+    - alert: CephVolumeFull
```

Comment:
ceph volume full doesn't make sense.  
on ceph nodes this partition is small.  all it containers is a partition to with mount points.  even if it was full, I am not sure it would affect anything.  

a better check would be if it is present and a partition/mount point.

Reply:
spoke with Rambabu & Shreni about getting a new ticket to monitor if a partition is mounted or not.

Reply:
As discussed with Craig, we have resolved it.